playground/components/Playground.js

```diff
@@ -35,21 +35,34 @@ class Playground extends React.Component {
 
   render() {
     return (
-      <div className="playgrund-app">
-        <div className="column">
-          <EditorComponent errors={this.state.parseResult.errors} />
+      <div>
+        <div className="header">
+        <div className="leftSide"><span>Attributes Kit Playground</span></div>
+        <div className="rightSide">
+          <span>
+            <a target="_blank" className="gitHubIco" href="https://github.com/apiaryio/attributes-kit">Attributes Kit Repository</a>
+          </span>
+          <span>
+            <a target="_blank" className="documentationIco" href="https://github.com/apiaryio/mson">Mson Documentation</a>
```

Comment:
Ok!
Keep in mind it has got “In Progress” label, so review it does not make much sense.